uint64 timestamp		# time since system start (microseconds)


float32    phi_d
float32    theta_d
float32[3] torque_sp
float32[3] q_erro
float32    phi
float32    theta
float32    psi
float32[3] w_esti
float32[3] delta_esti_w
float32    delta_deri_esti_wr
float32    w_deri_esti_r
float32    u_w

float32    test1
float32    test2
float32    test3
float32    test4
float32    test5

